### DISCUSSION TITLE
posts/2023/06/brothers-and-sisters/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#8] [General]
# posts/2023/06/brothers-and-sisters/

在哥哥姐姐的眼里，我们也是永远长不大的小孩。

https://guozheng.netlify.app/posts/2023/06/brothers-and-sisters/

**Comment by CyrusYip** [comment]
有这样的哥哥姐姐真幸福。

> 而让我最不能忘的，是无论与我毫无血缘关系的，还是作为堂系的哥哥姐姐，都视我如己出。

「视如己出」的意思是「当作自己亲生小孩看待」。你这里大概想说「把我当亲生兄弟」？

> 人们常常说，在爸爸妈妈眼里，我们永远都是长不大的小孩。其实在哥哥姐姐的眼里或许也是一样的。

我是独生子。我的哥哥姐姐有自己的小孩后我就不是那个最受关注的弟弟啦，当时还感觉有点惆怅。羡慕还在当小孩的你。

**Reply by residualsun1** [reply]
哈哈“视为己出”确实用错了，阅读和写作少了果然有大问题，很多时候词不达意，我就会靠感觉出来的词汇“将就”一下。不过不丢人！被指出问题就改～

我有一个亲哥哥，他现在工作啦，但我还在念书。在深圳的时候，我确实明显感觉出大家都会更加把关注我，吃饭的时候会一直往我这边照顾（夹菜、加饭啥的）。不过，虽然哥哥姐姐有了小孩后难免减少对弟弟的关注，但关键时刻还是能让弟弟鼻子一酸。我姐姐的小孩已经一岁多了，但我哥哥结婚的时候她全程张罗，和我们几个弟弟待在一起的时候，无论什么时候也都是惦记着带我们去好吃好喝。
